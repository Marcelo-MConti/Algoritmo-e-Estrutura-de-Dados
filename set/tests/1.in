0
6 12
-1 33 10 29 28 45
-1 2 45 99 37 66 111 44 -10 10 11 80
2
