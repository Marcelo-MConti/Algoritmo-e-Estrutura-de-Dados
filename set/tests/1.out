-10, -1, 2, 10, 11, 28, 29, 33, 37, 44, 45, 66, 80, 99, 111, 
